src/diffusers/models/modeling_utils.py

```diff
@@ -433,11 +437,11 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         # Load model
 
         model_file = None
-        if is_safetensors_available():
-            try:
+        if from_flax:
+            if is_flax_available():
```

Comment:
```suggestion
```

That's actually not needed as if it's not available it will already fail nicely before :-)